<ui version="4.0" >
 <class>STGlobalView</class>
 <widget class="QWidget" name="STGlobalView" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" >
   <property name="spacing" >
    <number>0</number>
   </property>
   <property name="margin" >
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="button_frame" >
     <property name="frameShape" >
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" >
      <property name="spacing" >
       <number>0</number>
      </property>
      <property name="margin" >
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout" >
        <item>
         <widget class="QToolButton" name="delete_button" >
          <property name="toolTip" >
           <string>Delete currently selected global parameter.</string>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
          <property name="icon" >
           <iconset resource="st4.qrc" >
            <normaloff>:/images/cross.png</normaloff>:/images/cross.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="up_button" >
          <property name="toolTip" >
           <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-size:8pt;">Move selected global parameter &lt;/span>&lt;span style=" font-size:8pt; font-weight:600;">up&lt;/span>&lt;span style=" font-size:8pt;"> in the list.&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
          <property name="icon" >
           <iconset resource="st4.qrc" >
            <normaloff>:/images/up.PNG</normaloff>:/images/up.PNG</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="down_button" >
          <property name="toolTip" >
           <string>&lt;!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 4.0//EN" "http://www.w3.org/TR/REC-html40/strict.dtd">
&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">Move selected global parameter &lt;span style=" font-weight:600;">down&lt;/span> in the list.&lt;/p>&lt;/body>&lt;/html></string>
          </property>
          <property name="text" >
           <string>...</string>
          </property>
          <property name="icon" >
           <iconset resource="st4.qrc" >
            <normaloff>:/images/down.PNG</normaloff>:/images/down.PNG</iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer" >
          <property name="orientation" >
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0" >
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="STGlobalViewTree" name="tree" >
     <column>
      <property name="text" >
       <string>1</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>STGlobalViewTree</class>
   <extends>QTreeWidget</extends>
   <header>stglobalview.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="st4.qrc" />
 </resources>
 <connections/>
</ui>
